.github/workflows/checks.yml

```diff
@@ -8,6 +8,7 @@ permissions:
 env:
   DEBUG: True
   NOTIFY_ENVIRONMENT: test
+  NEW_RELIC_ENVIRONMENT: test
```

Comment:
I'm probably misunderstanding things, but what does this env var do here? If it's here, do we also need the config file and license key env vars?

Reply:
`NEW_RELIC_ENVIRONMENT` sets which of the `[newrelic:<NEW_RELIC_ENVIRONMENT>]` blocks to activate within `newrelic.ini`

Reply:
Though... that won't actually do anything because I'm missing the config file value. I'll add the config file. We don't need the license key since test tells it not to report any data so I'll leave that off to avoid leaking it accidentally.

Reply:
Cool, good call. Silly question, though: what is New Relic actually doing in the test environment. I looked but couldn't immediately figure out where it comes into play.

Reply:
Okay, we discussed and including the env vars makes sure that we have a config that explicitly _doesn't_ send data to New Relic, which is what we want.